index.html

```diff
@@ -251,67 +222,59 @@ <h3 id="matters-considered-by-the-placement-rules">Matters considered by the pla
 <ol>
   <li id="l20200529007">Ruby is used to display the reading or the meaning of the base characters.
     Therefore, the number one priority here is to avoid misreadings.
-    Specifically, the ruby string which protrudes from the base character 
-    string is not allowed to be laid over the characters preceding or 
-    following, whether it is a Kanji or Kana character. 
+    Specifically, this method does not allow a ruby annotation which is wider than its base text to overhang the characters preceding or 
+    following, whether they are kanji or kana characters. 
 
-    <aside class=note title="Protruding over surrounding characters" id="n20200529003">
-      The main placement method defined in JIS X 4051 [[JISX4051]]
+    <aside class=note title="Overhanging surrounding characters" id="n20200529003">
+      The main placement method defined in [[JISX4051]]
       allows some amount of overhang over the preceding and following base characters,
-      but recognizes the method defined here as an allowed variant.
+      but also recognizes the method defined here as an allowed variant.
     </aside>
   </li>
   <li id="l20200529008">The method is agnostic to horizontal vs vertical writing,
     and will use the same logic in either case.
     Specifically, the center of the ruby string and of the base character 
     string are aligned in the inline direction for mono-ruby. 
-    </li>
-  <li id="l20200529009">Two-step processing method is taken. 
-    In the first step, processing of layout only considers about the ruby 
-    string and the base character string (collectively call both of them as 
-    the ruby block in this document), to decide relative position of the ruby 
-    string and the ruby base character string. 
-    In the second step, processing of layout decides a position of the ruby 
-    base character string in a line, with consideration of preceding and 
+</li>
+  <li id="l20200529009">Processing is done in two steps. 
+    In the first step, processing of layout only considers  the relative positions of the ruby 
+    annotation and its base text (we refer to these, collectively, in this document as the ruby-block). 
+    In the second step  layout processing decides the position of the ruby 
```

Comment:
propose to have "," between step and layout
```suggestion
    In the second step, layout processing decides the position of the ruby 
```

Reply:
Fixed.